browser/brave_profile_prefs.cc

```diff
@@ -231,6 +233,12 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterStringPref(kBinanceRefreshToken, "");
 #endif
 
+  // Gemini widget
+#if BUILDFLAG(GEMINI_ENABLED)
+  registry->RegisterStringPref(kGeminiAccessToken, "");
+  registry->RegisterStringPref(kGeminiRefreshToken, "");
```

Comment:
Looks like we're not using this yet, we'll need to use it to get a new access token when the access token is invalid. Perhaps they can give you an endpoint to call to manually invalidate a token so we can test to make sure we have it right?  This would be in a followup.

Reply:
this should go in static `GeminiService::RegisterProfilePrefs`